src/app/contexts/RequestContext/getOriginContext/index.js

```diff
@@ -5,9 +5,7 @@ const getOriginContext = bbcOrigin => {
   if (bbcOrigin) {
     origin = bbcOrigin;
   } else if (process && process.env && process.env.APP_ENV === 'local') {
-    // For local dev, set origin to Test
-    // Ensures analytics are sent to Test bucket
-    origin = 'https://www.test.bbc.co.uk';
+    origin = 'http://localhost.bbc.com:7080';
```

Comment:
I'd rather we used `process.env.SIMORGH_BASE_URL` unless anyone can suggest any reason not to, to avoid repeating this value multiple times throughout the codebase.